<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelEditor</class>
 <widget class="QMainWindow" name="LevelEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LevelEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="objectPalette">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>1031</width>
      <height>61</height>
     </rect>
    </property>
    <property name="flow">
     <enum>QListView::LeftToRight</enum>
    </property>
   </widget>
   <widget class="CustomGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>931</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="modeList">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>91</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>customgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
